README.md

```diff
@@ -312,6 +312,8 @@ Currently only a `:naive` form of parallelism is supported, which just uses
 `pmap` to run the linters over your namespaces. You can specify
 `:parallelism?` in your options map, currently `:none` and `:naive` are valid options.
 
+> :parallelism? performs AST analysis/evaluation in parallel and might cause issues.
```

Comment:
No urgency, but personally I think we should consider removing `:parallelism?`.

`require`ing clojure core in parallel in a not particularly structured manner seems a recipe for heisenbugs.

There are other possible lines of improvements (refactor-nrepl style AST caching, deriving workloads out of git branch diffs, etc)